code/controllers/Processes/spacedrift.dm

```diff
@@ -0,0 +1,57 @@
+#define MAX_THROWING_DIST 512 // 2 z-levels on default width
+#define MAX_TICKS_TO_MAKE_UP 3 //how many missed ticks will we attempt to make up for this run.
```

Comment:
Honestly I think I copypasted the throwing controller file and changed it to spacedrift, fixing